spk/ffmpeg/Makefile

```diff
@@ -1,16 +1,18 @@
 SPK_NAME = ffmpeg
-SPK_VERS = 2.1.1
-SPK_REV = 1
+SPK_VERS = 2.8.5
+SPK_REV = 4
 SPK_ICON = src/ffmpeg.png
 DSM_UI_DIR = app
-CHANGELOG = update ffmpeg
+CHANGELOG = "1. update ffmpeg to 2.8.5<br>2. added 13 new libs/codecs<br>3. support for yasm added"
+
 
 DEPENDS = cross/$(SPK_NAME)
 
-MAINTAINER = cytec
+MAINTAINER = SynoCommunity
```

Comment:
You can add yourself as maintainer nowadays, I mentioned that in an issue somewhere.